cmd/plume/release.go

```diff
@@ -40,21 +39,62 @@ var (
 	cmdRelease    = &cobra.Command{
 		Use:   "release [options]",
 		Short: "Publish a new CoreOS release.",
-		Run:   runRelease,
+		RunE:  runRelease,
```

Comment:
Why is this being switched to `RunE`? Given `runRelease` is the top level function it should be able to utilize `plog.Fatal` as it hasn't actually spawned any awaiting to be cleaned up once the error has bubbled up.